bin/main.rb

```diff
@@ -0,0 +1,31 @@
+require_relative '../lib/game_logic'
+
+puts "\n" + '*' * 50
```

Comment:
👉 this part of the code can be housed in another file and imported into main. Main should only serve as an entry point for your app execution🙂